pkgs/development/python-modules/mpi4py/default.nix

```diff
@@ -46,14 +46,14 @@ buildPythonPackage rec {
     export OMPI_MCA_rmaps_base_oversubscribe=yes
   '';
 
-  setupPyBuildFlags = ["--mpicc=${mpi}/bin/mpicc"];
+  setupPyBuildFlags = [ "--mpicc=${mpi}/bin/mpicc" "build_src --force" ];
```

Comment:
Can you explain what `build_src --force does`? Maybe a comment inside the expression would be nice.